src/package_io/input_parser.star

```diff
@@ -15,7 +15,7 @@ def get_default_input_args():
         "ephemeral_ports": True,
         "avalanchego_image": DEFAULT_AVALANCHEGO_IMAGE,
         "node_config": default_node_cfg,
-        "node_count": 5,
+        "node_count": 2,
```

Comment:
can we adda `fail` to fail if people set a node count of less than 2?

Reply:
Yes, done that now.